OAuthSDK/YMAPIClient.h

```diff
@@ -23,20 +23,27 @@
 #import <Foundation/Foundation.h>
 #import "AFNetworking.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 extern NSString * const YMBaseURL;
 
+typedef void(^successCallback)(id __nullable responseObject);
+typedef void(^failureStatusCodeCallback)(NSInteger statusCode, NSError *error);
+typedef void(^failureCallback)(NSError *error);
+
 /**
  Represents an object that contains a queue of HTTP operations.
  At the moment, this is essentially a lightweight wrapper around AFHTTPSessionManager.
  */
 @interface YMAPIClient : NSObject
-@property (nonatomic, copy) NSString *authToken;
+
+@property (nonatomic, copy, nullable) NSString *authToken;
 
 /**
  Default initializer.
  @param authToken The OAuth token.
  */
-- (id)initWithAuthToken:(NSString *)authToken;
+- (instancetype)initWithAuthToken:(nullable NSString *)authToken;
```

Comment:
I got the impression Apple wants you to use the non-underscored version where possible:

_You can use __nullable and __nonnull almost anywhere you can use the normal C const keyword, though of course they have to apply to a pointer type. However, in the common case there’s a much nicer way to write these annotations: within method declarations you can use the non-underscored forms nullable and nonnull immediately after an open parenthesis, as long as the type is a simple object or block pointer._

This came from: https://developer.apple.com/swift/blog/?id=25